### PR TITLE
[Promote] Potential Masquerading as Communication Apps

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Promotes from BBR to DR, have been running for the last 2 months with minor FPs, the current query has only 2 hits on telemetry related to testing.

### CODE REVIEW

File: rules/windows/defense_evasion_masquerading_communication_apps.toml [hunk 1]
```diff
@@ -18,12 +17,11 @@ index = ["logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Potential Masquerading as Communication Apps"
-risk_score = 21
+risk_score = 47
 rule_id = "c9482bfa-a553-4226-8ea2-4959bd4f7923"
-severity = "low"
+severity = "medium"
 tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Rule Type: BBR", "Data Source: Elastic Defend"]
```
Comment (w0rk3r) [inline rules/windows/defense_evasion_masquerading_communication_apps.toml:23]:
```suggestion
tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Data Source: Elastic Defend"]
```

File: rules/windows/defense_evasion_masquerading_communication_apps.toml [hunk 2]
```diff
@@ -45,7 +43,7 @@ process where host.os.type == "windows" and
 
     /* Teams */
     (process.name : "Teams.exe" and not
-      (process.code_signature.subject_name == "Microsoft Corporation" and process.code_signature.trusted == true)
+      (process.code_signature.subject_name in ("Microsoft Corporation", "Microsoft Corporation") and process.code_signature.trusted == true)
```
Comment (imays11) [inline rules/windows/defense_evasion_masquerading_communication_apps.toml:46]: What's the difference between these 2 values?

Comment (Aegrah) [reply rules/windows/defense_evasion_masquerading_communication_apps.toml:46]: ++

**Review by Aegrah — APPROVED** [review]
After adding additional exclusion via the `in` statement, it LGTM

File: rules/windows/defense_evasion_masquerading_communication_apps.toml (hunk 2 above)
Comment (w0rk3r) [reply rules/windows/defense_evasion_masquerading_communication_apps.toml]: Nice catch, there is no difference 😅

**Review by terrancedejesus — APPROVED** [review]

**Pull request merged**